roles/confluent.ksql/tasks/rbac.yml

```diff
@@ -85,6 +89,11 @@
             "resourceType":"Group",
             "name":"_confluent-ksql-{{ksql.properties['ksql.service.id']}}",
             "patternType":"PREFIXED"
+          },
+          {
+            "resourceType":"Topic",
+            "name":"{{ksql.properties['ksql.service.id']}}processing_log",
+            "patternType":"LITERAL"
```

Comment:
is this block necessary? looks like the rolebinding is conditionally created below?